src/components/Card/Card.tsx

```diff
@@ -0,0 +1,15 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { FC, ReactChild } from "react";
+
+export type CardProps = {
+    children?: ReactChild;
+};
+
+export const Card: FC<CardProps> = ({ children }) => {
+    return (
+        <div data-test-id="card" className="tw-w-full tw-bg-white tw-border tw-border-black-10 tw-rounded">
```

Comment:
@thisguyscode should the card also include the 12px padding or does that belong to the content?

Reply:
Jumping in here with regards to the reason why I currently left it out - as some of our use cases are going to have a Divider that stretches across the entire card width, we'd either need to start overriding the original padding or make it optional. But I'd be happy to change it however we decide :)

Reply:
@jonasbuechel @marguslant0 

0 padding should definitely be an option. If we also want padded options, we can provide it as a prop. I'd suggest that 0 padding remains the default.

Reply:
Marking it as resolved, then, if you agree, @jonasbuechel?